src/ActionModal/components/DelegationModal.vue

```diff
@@ -81,6 +81,7 @@
     </TmFormGroup>
 
     <TmFormGroup
+      v-if="!(session.addressRole === `stash`)"
```

Comment:
```suggestion
      v-if="!(session.addressRole === `stash`)"
```
```suggestion
      v-if="session.addressRole !== `stash`"
```

Reply:
But you were the one saying that positive was always easier to understand xD

Reply:
that would be if you create a variable `notAStash` vs a variable `isStash`

Reply:
Done